nucliadb_node/src/writer/grpc_driver.rs

```diff
@@ -237,7 +234,16 @@ impl NodeWriter for NodeWriterGRPCDriver {
                 status.detail = "Success!".to_string();
                 Ok(tonic::Response::new(status))
             }
-            Err(error) => Err(tonic::Status::internal(error.to_string())),
+            Err(error) => {
+                let status = OpStatus {
+                    status: op_status::Status::Error as i32,
+                    detail: error.to_string(),
+                    field_count: 0_u64,
+                    shard_id,
+                    ..Default::default()
+                };
+                Ok(tonic::Response::new(status))
+            }
```

Comment:
Maybe using one expression simplifies the code:
```rust
            Err(error) => Ok(tonic::Response::new(OpStatus {
                status: op_status::Status::Error as i32,
                detail: error.to_string(),
                field_count: 0_u64,
                shard_id,
                ..Default::default()
            })),
```

Reply:
Same